draft-ietf-rtcweb-jsep.xml

```diff
@@ -2581,6 +2586,11 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                         <t>The "a=rtcp-mux" line MUST only be added if
                         present in the most recent answer.</t>
 
+			<t>An "a=rtcp-mux-only" line, as specified in
```

Comment:
I suppose there are two cases here:
1) other side knows about mux-only: then the answer should always have it, and the 'if in most recent answer' text is fine.
2) other side doesn't know about mux-only: then there's no point in including it, and the text is also fine.

If the remote side doesn't support rtcp-mux, of course, things will blow up, but that's already been accounted for elsewhere.